README.md

```diff
@@ -916,7 +971,7 @@ If `--use-remote-relative-includes` option is specified, every [local include](#
 
 `--use-remote-relative-includes` option does not affect includes with [absolute remote paths](#include).
 
-**Note** In the current Builder version `--use-remote-relative-includes` option affects includes mentioned in remote source files from GitHub only.
+**Note** In the current Builder version `--use-remote-relative-includes` option affects includes mentioned in remote source files from GitHub and Bitbucket Server only.
```

Comment:
Remote relative includes, to work correctly, requires all Builder `@include`'s to be referenced by `__PATH__`. Is that covered in the documentation?

Reply:
At least, I think the `__PATH__` is still required for remote relative includes to work, unless some of the updates to the readers make that not necessary? I saw some updates around the path, but wasn't 100% sure what they were doing/fixing.

Reply:
No, the "remote relative includes" feature is exactly the way to turn simple `@include`'s like `@include "b.nut"` into `@include __PATH__ + "b.nut"` **without** adding the `__PATH__ + ` prefix into the code.
If you use `__PATH__`, you don't need this feature.

Reply:
Hey @ragrus-nbl, can you add a test case to test another example of the `remote-relative-include` feature?

I see for `spec/GithubReader.spec.js`, there is a test that reads the example from the electricimp repo and compares it to the built file from the local repo. But, I think we need a little more test coverage (and that goes for BBServer, too, but the test can be the same for both).

The behavior we expect is that, for remote relative includes, the `__PATH__` variable should not be necessary, and the `@include`s **should always default to the repo root (as they do locally) for the search path**.

The file structure (in the remote) should look like the following:

```
-root
   |
   -spec
        |
        -fixtures
             |
             -sample-new
                   |
                   - rootFile.nut
                   - subFolder1
                         |
                         - subFolder1File1.nut
                   - subFolder2
                         |
                         - subFolder2File1.nut
```

Contents of `subFolder1File1.nut`:

```squirrel
@include "rootFile.nut" //notice that the path here assumes that the root of the repo is the default search path
```

Contents of `rootFile.nut`:

```squirrel
@include "subFolder2/subFolder2File1.nut"
```

Contents of `subFolder2File1.nut`:

```squirrel
server.log("subFolder2File1.nut")
```

The test case should perform a `@include` of all 3 paths (example below just shows the path to `subFolder1File1.nut`:

```js
    remote = reader.read('github:electricimp/Builder/spec/fixtures/sample-new/subFolder1/subFolder1File1.nut@master');
```

And the output should be the same each time. The real test here is ensuring that the `@include` in `subFolder1File1.nut` doesn't need to be `@include "./../rootFile.nut` or `@include __PATH__ + "/../rootFile.nut" (which is necessary with the version we are using now). Basically, we don't want the `__PATH__` variable to be necessary in any libraries.

Reply:
Hi @mm-gmbd, I would suggest doing this within EMCB-106 since this task (EMCB-94) is not about the `remote-relative-include` feature. All the changes to the `remote-relative-include` feature in this PR were just merged from https://github.com/deldrid1/Builder/tree/hotfix/remote-relative-includes and nothing more.
Does this make sense for you?

Reply:
Yes, that makes sense. Please add to the scope of EMCB-106. Thanks!